js/scripts.js

```diff
@@ -67,6 +67,17 @@ function updateInputs(clone, newNum) {
     });
 }
 
+// Function to reset the text input when creating new form
+function resetInputText (newElemCloned) {
+    let initialInputs = newElemCloned.querySelectorAll("input[type=text]");
+    for (let i = 0; i < initialInputs.length; i++){
+        console.log(initialInputs[0].value);
```

Comment:
Should remove or comment out this log (console.log blocks the js thread)

Reply:
oh i forgot to do that, will do!